src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -241,19 +241,31 @@ public static string DoYouWantToDeleteDirectory(string newDirectoryName)
          return $"Do you want to delete the directory '{newDirectoryName}' and continue?";
       }
 
+      private static void appendListItem(string listItem, StringBuilder sb)
```

Comment:
Should not we make those methods public somewhere so that then can be used from other components in the future?